inc/class-addon-manager.php

```diff
@@ -266,11 +270,78 @@ protected function convert_subscription_to_plugin( $subscription ) {
 			'download_link' => $subscription->product->download,
 			'package'       => $subscription->product->download,
 			'sections'      => [
-				'changelog' => $subscription->product->changelog,
+				'changelog' => $changelog,
+				'support'   => $this->get_support_section(),
+			],
+			'icons'         => [
+				'2x' => $this->get_icon( $subscription->product->slug ),
 			],
+			'banners'       => $this->get_banners( $subscription->product->slug ),
+			'tested'        => YOAST_SEO_WP_TESTED,
+			'requires'      => YOAST_SEO_WP_REQUIRED,
+			'requires_php'  => YOAST_SEO_REQUIRED,
```

Comment:
Assuming this should be `YOAST_SEO_PHP_REQUIRED`. Also, tests fail because they still use the old constant names